app/editor/components/Styles.ts

```diff
@@ -1287,6 +1287,10 @@ const EditorStyles = styled.div<{
     .page-break {
       opacity: 0;
     }
+    
+    button.show-source-button {
```

Comment:
Correct approach, but please include in the block of other hidden elements above starting L1275

Reply:
@tommoor Of course, I didn't see that. I committed the new file